kernel/search/plugins/ezsearchengine/ezsearchengine.php

```diff
@@ -68,7 +68,6 @@ public function addObject( $contentObject, $commit = true )
         $placement = 0;
         $previousWord = '';
 
-        eZContentObject::recursionProtectionStart();
```

Comment:
wondering if we might need to keep these in, in case custom relation field types stiull use it.

or will it cause problems?

Reply:
We can (and should) keep those, you are right.